ipld/read_test.go

```diff
@@ -231,3 +234,66 @@ func removeRandShares(data [][]byte, d int) [][]byte {
 	}
 	return data
 }
+
+func TestGetSharesByNamespace(t *testing.T) {
```

Comment:
Shouldn't this be (?): 
```suggestion
func Test GetLeavesByNamespace(t *testing.T) {
```